src/transformers/models/layoutlm/__init__.py

```diff
@@ -54,6 +67,17 @@
             LayoutLMForTokenClassification,
             LayoutLMModel,
         )
+    if is_tf_available():
+        from .modeling_tf_layoutlm import (
+            TF_LAYOUTLM_PRETRAINED_MODEL_ARCHIVE_LIST,
+            TFLayoutLMEmbeddings,
```

Comment:
```suggestion
```

No need for this one (it's also not in the other aprt of the init, not is it in the main init). People can go import it in the right module.

Reply:
fixed [here](https://github.com/huggingface/transformers/pull/10636/commits/3bee70daf2a71066335d360761ce5c7bb432500a)